components/mintmaker/base/mintmaker-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: renovate-config
+  namespace: mintmaker
+data:
+  renovate.json: |
+    module.exports = {
+      "onboarding": false,
+      "requireConfig": "ignored",
+      "platform": "github",
```

Comment:
The controller will patch the `platform` accordingly, so it's unnecessary to include this.